evasquez26/angular4-startup
# Contact Us Form

## Story / Task


see [ONE-172](https://bixlabs.atlassian.net/browse/ONE-172)


## What this PR does?
Added a contact us form. Creates a ticket for the user question.


### Before 

There wasn't a contact us form.

### After

There is a contact form.

### How to test / reproduce

- Run the app.
- On the footer, select 'Contact us' 
- Fill all the fields. All of them should be required.
- Hit 'Send' button.
- A success message should appear.
- If a user is logged in, when said user enters the contact us page, the user's name should be preloaded in the 'Name' field.

### TODOs
N/A

### Related PRs
N/A

## Migrations

N/A


-----------
Thanks!
